styles/variation-sherbet.json

```diff
@@ -0,0 +1,248 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 2,
+	"title": "Variation Sherbet",
+	"settings": {
+		"color": {
+			"duotone": [
+				{
+					"colors": [
+						"rgb(81, 82, 213)",
+						"rgb(121, 238, 196)",
+						"rgb(236, 249, 87)",
+						"rgb(247, 69, 204)"
+					],
+					"name": "Default filter",
+					"slug": "default-filter"
+				}
+			],
+			"gradients": [
+				{
+					"gradient": "linear-gradient(135deg, var(--wp--preset--color--primary) 0%, var(--wp--preset--color--secondary) 50%, var(--wp--preset--color--tertiary) 100%)",
+					"name": "Primary to Secondary to Tertiary",
+					"slug": "primary-secondary-tertiary"
+				}
+			],
+			"palette": [
+				{
+					"color": "#FFFFFF",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#000000",
+					"name": "Contrast",
+					"slug": "contrast"
+				},
+				{
+					"color": "#FFCCFF",
+					"name": "Primary",
+					"slug": "primary"
+				},
+				{
+					"color": "#FFFFCC",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#CCFFFF",
+					"name": "Tertiary",
+					"slug": "tertiary"
+				}
+			]
+		},
+		"typography": {
+			"fontFamilies": [
```

Comment:
We don't need to redefine the font families in the variation file, so we can remove the 'fontFamilies' array.